packages/api/Dockerfile

```diff
@@ -1,30 +1,13 @@
-FROM	node:16	as	builder
-WORKDIR	/app
+FROM node:18 as builder
```

Comment:
i was contemplating if we switch it to `node:lts` but our builds from caches might not pick up when lts switches from 18 to 20 (and also dependabot won't know what to do)

Reply:
I hit some problem when jumping to 20, comfortable with 18 for now